### PR TITLE
chore: address v1 comments

### PR DESCRIPTION
**Pull request by obi-ewan**
Implement changes after V1 review:
* [x] refactor: rename slug to code 
* [x] feat: active/deactivate instructions for participant
* [x] refactor: ensure authority field is filterable
* [x] feat: add restriction, only category owner can create participants
* [x] feat: add close account instructions
* [x] chore: add vanity address `mpeq1VkcGKDWS7ELiQJ2diRPX9vHQb27FruqcoxhPqF`

### CODE REVIEW

File: admin/create_event.ts [hunk 1]
```diff
@@ -4,26 +4,26 @@ import { findEventPda, getProgram, getAnchorProvider } from "./util";
 
 if (process.argv.length < 6) {
   console.log(
-    "Usage: yarn run createEvent <SLUG> <NAME> <START_TIME_MS> <PARTICIPANTS_ARRAY> [ORACLE] [REFERENCE]",
+    "Usage: yarn run createEvent <CODE> <NAME> <START_TIME_MS> <PARTICIPANTS_ARRAY> [ORACLE] [REFERENCE]",
```
Comment (eoin-betdex) [inline admin/create_event.ts]: ORACLE and REFERENCE are no longer required throughout.

File: programs/protocol_event/src/error.rs [hunk 2]
```diff
@@ -8,4 +8,6 @@ pub enum EventError {
     MaxParticipantsExceeded,
     #[msg("Attempted to add invalid event participants.")]
     InvalidEventParticipants,
+    #[msg("Authority mismatch.")]
```
Comment (eoin-betdex) [inline programs/protocol_event/src/error.rs]: NP I know we have them here but I don't think anchor error messages are expected to end with a period. As I think we end up with `..` at the end of the output when we do.

File: jest.config.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 module.exports = {
-    verbose: false,
+    verbose: true,
```
Comment (eoin-betdex) [inline jest.config.js]: 🔕

File: programs/protocol_event/src/context.rs [hunk 4]
```diff
@@ -141,3 +141,57 @@ pub struct UpdateParticipant<'info> {
...
+pub struct CloseEvent<'info> {
+    #[account(
+        mut,
+        has_one = authority,
+        close = payer,
+    )]
+    pub event: Account<'info, Event>,
+    pub authority: Signer<'info>,
+    #[account(mut)]
+    pub payer: SystemAccount<'info>,
```
Comment (eoin-betdex) [inline programs/protocol_event/src/context.rs:158]: Should there be a `has_one` constraint against `event.payer` here or something similar?

Comment (eoin-betdex) [reply programs/protocol_event/src/context.rs:158]: Similar comment for all structs below.

**Review by eoin-betdex — CHANGES_REQUESTED** [review]
We should probably revert the change to verbosity in the jest config, but apart from that LGTM. A few small nitpicks but nothing major.

File: programs/protocol_event/src/context.rs (hunk 4 above)
Comment (obi-ewan) [reply programs/protocol_event/src/context.rs:158]: ah good catch

File: admin/create_event.ts (hunk 1 above)
Comment (obi-ewan) [reply admin/create_event.ts]: The admin scripts need some love, I will fix this one but add a task to create/update/delete them for testing in dev

File: programs/protocol_event/src/context.rs (hunk 4 above)
Comment (obi-ewan) [reply programs/protocol_event/src/context.rs:158]: I've added a `has_one = payer` constraint to the account being closed

**Review by pawel-betdex — APPROVED** [review]
damn slugs :+1:

**Review by eoin-betdex — APPROVED** [review]

**Pull request merged**